pages/about.html

```diff
@@ -0,0 +1,126 @@
+    <main>
+      <section class="hero">
+        <nav class="mobile">
+          <div class="burger">
+            <span></span>
+            <span></span>
+            <span></span>
+          </div>
+          <div class="mobile-menu">
+            <ul>
+              <li><a href="./index.html">Home</a></li>
+              <li><a href="./about.html">About</a></li>
+              <li>Programe</li>
+              <li>Join</li>
+              <li>Sponser</li>
+              <li>News</li>
+              <li><a class="highlight" href="">Microsoft</a></li>
+            </ul>
+            <div class="close">
+              <img src="../images/home/XIcon.svg" alt="" />
+            </div>
+          </div>
+        </nav>
+        <div class="inner">
+          <h1>
+            creating commons <br />
+            global summit 2022
+          </h1>
+          <div class="line"></div>
+          <div class="box">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+              Consequatur molestiae <br />
+              distinctio placeat. Voluptatem provident ab nesciunt doloribus aut
+              obcaecati doloremque quod beatae <br />
+              accusantium vero atque pariatur corporis dolores est eaque, quia
+              non quidem, blanditiis autem maiores culpa inventore tenetur
+              voluptates saepe. Rerum labore voluptatum, aliquid accusamus iusto
+              et, dolore mollitia provident natus distinctio minima aut quasi,
+              repudiandae quisquam dicta. Perferendis?
+            </p>
+          </div>
+          <p class="contact">
+            Please contact us per email for any further questions about CC
+            Global Summit 2022
+          </p>
+          <a href="#">ccsummit2022@cckorea.org</a>
+        </div>
+      </section>
+      <section class="creative">
+        <h3>Creative Commons Global Summit 2022 Logo</h3>
+        <div class="line"></div>
+        <div class="holder">
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Velit
+            earum, maiores, sequi consectetur repudiandae quos dolor asperiores,
+            voluptas natus eveniet consequuntur magni veniam voluptates quae
+          </p>
+          <div class="logo-box">
+            <img src="../images/microsoft-logo.svg" alt="" />
+          </div>
+        </div>
+      </section>
+      <section class="prev">
+        <h3>See the past CC global Summits</h3>
+        <div class="line"></div>
+        <p>
+          Take a lookt at the last two CC Global Summits which took place in
+          Buenos Aires and in Warsaw.
+        </p>
+        <article class="group">
+          <div></div>
+          <div class="bg-2011"></div>
+        </article>
+      </section>
+    </main>
+    <footer class="about-footer">
+      <div>
+        <img src="../images/microsoft-logo.svg" alt="" />
+        <p>© Microsoft 2022</p>
+      </div>
+    </footer>
```

Comment:
- [ ] it would be better if you make the footer in the about page (desktop version) black as in the requirements in order to match the design 
- Yours 
<img width="1433" alt="Screen Shot 2022-10-20 at 16 10 02" src="https://user-images.githubusercontent.com/61790587/196987807-4e5bb700-7a57-470a-be19-05cbdd03bfe2.png">
- Design 
<img width="877" alt="Screen Shot 2022-10-20 at 16 10 35" src="https://user-images.githubusercontent.com/61790587/196987818-45c9188a-d3ad-4476-9619-8cfbcb5a98f9.png">